rcl_action/test/rcl_action/test_action_interaction.cpp

```diff
@@ -0,0 +1,906 @@
+// Copyright 2018 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#include <gtest/gtest.h>
+
+#include "rcl_action/action_client.h"
+#include "rcl_action/action_server.h"
+#include "rcl_action/wait.h"
+
+#include "rcl/error_handling.h"
+#include "rcl/rcl.h"
+
+#include "rosidl_generator_c/primitives_sequence_functions.h"
+
+#include "test_msgs/action/fibonacci.h"
+
+#ifdef RMW_IMPLEMENTATION
```

Comment:
I guess we'll need some cmake logic for testing multiple RMW implementations. Perhaps we can rebase after #351 is merged.

Reply:
you're right, just did as suggested. tests are now running against multiple RWM implementations. thanks!